testsuite/setup.cfg

```diff
@@ -18,6 +18,8 @@ filterwarnings=
     ignore:Reader has no dt information, set to 1.0 ps
     # NamedStream warnings
     ignore:Constructed NamedStream:RuntimeWarning
+    # Stride block warnings 
+    ignore:Stride of block:UserWarning
```

Comment:
We specifically do not want to do a block ignore for UserWarnings across our tests. The aim of the issue is to have specific filterwarnings on the pytng tests for warnings we specifically expect that each test should raise.